src/backend/apps/utils/middleware.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from pprint import pformat
+
+
+logger = logging.getLogger(__name__)
+
+
+class RequestDataLoggingMiddleware:
```

Comment:
handy, nice, may want to move this to django-ckc so we can update it across projects ez